### ISSUE TITLE
Extraire un seul fichier d'un tar

### ISSUE THREAD

**Issue by benoistlaurent** [DU-Bii/module-1-Environnement-Unix#11]
Ça serait sans doute pas honteux de présenter ça l'année prochaine...

Votre avis ?

**Comment by pierrepo** [comment]
Non, c'est pas scandaleux. Mais il faut au préalable introduire l'option qui permet d'afficher le contenu d'un tar, sans extraire les fichiers.
